####### The service to set the current robot position

### Service inputs

# The current robot pose as a 2D coordinate to be stored.
Point pose

---

### No service outputs
